content/en/docs/components/notebooks/quickstart-guide.md

```diff
@@ -0,0 +1,113 @@
++++
+title = "Quickstart Guide"
+description = "Getting started with Kubeflow Notebooks"
+weight = 10
+                    
++++
+
+## Summary
+
+1. Install Kubeflow by following [Getting Started - Installing Kubeflow](/docs/started/installing-kubeflow/).
+2. Open the Kubeflow [Central Dashboard](/docs/components/central-dash/) in your browser.
+3. Click __"Notebooks"__ in the left-hand panel.
+4. Click __"New Server"__ to create a new notebook server.
+5. Specify the configs for your notebook server.
+6. Click  __"CONNECT"__ once the notebook has been provisioned
+
+## Detailed Steps
+
+1. Open the Kubeflow [Central Dashboard](/docs/components/central-dash/) in your browser.
+
+2. Select a Namespace:
+    - Click the namespace dropdown to see the list of available namespaces.
+    - Choose the namespace that corresponds to your Kubeflow Profile.
+      (See the page on [multi-user isolation](/docs/components/multi-tenancy/) for more information about Profiles.)
+
+   <img src="/docs/images/notebooks-namespace.png"
+   alt="Selecting a Kubeflow namespace"
+   class="mt-3 mb-3 border border-info rounded">
+
+3. Click __"Notebook Servers"__ in the left-hand panel:
+
+   <img src="/docs/images/jupyterlink.png"
+   alt="Opening notebooks from the Kubeflow UI"
+   class="mt-3 mb-3 border border-info rounded">
+
+4. Click __"New Server"__ on the __"Notebook Servers"__ page:
+
+   <img src="/docs/images/add-notebook-server.png"
+   alt="The Kubeflow notebook servers page"
+   class="mt-3 mb-3 border border-info rounded">
+
+5. Enter a __"Name"__ for your notebook server.
+    - The name can include letters and numbers, but no spaces.
+    - For example, `my-first-notebook`.
+
+   <img src="/docs/images/new-notebook-server.png"
+   alt="Form for adding a Kubeflow notebook server"
+   class="mt-3 mb-3 border border-info rounded">
+
+6. Select a Docker __"Image"__ for your notebook server
+    - __Custom image__: If you select the custom option, you must specify a Docker image in  the form `registry/image:tag`.
+      (See the guide on [container images](/docs/components/notebooks/container-images/).)
+    - __Standard image__: Click the __"Image"__ dropdown menu to see the list of available images.
+      (You can choose from the list configured by your Kubeflow administrator)
+
+7. Specify the amount of __"CPU"__ that your notebook server will request.
+
+8. Specify the amount of __"RAM"__ that your notebook server will request.
+
+9. Specify a __"workspace volume"__ to be mounted as a PVC Volume on your home folder.
+
+10. *(Optional)* Specify one or more __"data volumes"__ to be mounted as a PVC Volumes.
+
+11. *(Optional)* Specify one or more additional __"configurations"__
```

Comment:
And one for configurations. Given the number of options, it's easy for people to get lost. 

I realize we want to focus users on the required steps, but I think these screenshots add clarity.